### ISSUE TITLE
Improvement: avoid logging confidential information

### ISSUE THREAD

**Issue by algra4** [cmdotcom/text-sdk-java#22]
Hello, 

By default, you log the response: 
https://github.com/cmdotcom/text-sdk-java/blob/faa2d5c5b242f4c5db374ae296289f935c6d8c90/src/main/java/com/cm/text/utils/HttpHelper.java#L38

So I see the confidential information in my log as the phone number
Example:
```json
{  "details": "Created 1 message(s)",  "errorCode": 0,  "messages": [    {      "to": "xxxxxxxx",      "status": "Accepted",      "reference": null,      "parts": 1,      "messageDetails": null,      "messageErrorCode": 0    }  ]}
```

Is it possible to set logging parameters according to level, for example, or simply remove the ```System.out.println(response.toString());``` ?

Kind regards

**Comment by michelmegens** [comment]
Thanks (again) for catching this. After some debate internally, we'll remove the log line.

**Comment by EnessenE** [comment]
If all goes well this will be released later today under version 2.1.0
